tutorial/stdlib.po

```diff
@@ -8,25 +8,25 @@ msgstr ""
 "Project-Id-Version: Python 3.6\n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2016-10-30 10:40+0100\n"
-"PO-Revision-Date: 2016-11-19 15:52+0100\n"
+"PO-Revision-Date: 2018-02-13 23:13+0100\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 
 #: ../Doc/tutorial/stdlib.rst:5
 msgid "Brief Tour of the Standard Library"
-msgstr "Survol de la Bibliothèque Standard"
+msgstr "Survol de la bibliothèque standard"
```

Comment:
There's a debate on capitalization of titles, but why not sticking to the "title case" here? If we decide not to do so, we should document it in the README to ensure consistency.

Reply:
http://jacques-andre.fr/faqtypo/lessons.pdf , §2.6 page 12
"en français, pas de majuscule initiale à tous les substantifs d’un titre"

Reply:
I don't know the guy, but let it be, it make sense. I'll document it in the README and find others.